packages/components/psammead-sitewide-links/src/Link/index.jsx

```diff
@@ -19,19 +19,21 @@ const StyleLinkText = styled.span`
   }
 `;
 
-const Link = ({ text, href, inline }) => (
-  <StyledLink inline={inline} href={href}>
+const Link = ({ text, href, inline, lang }) => (
+  <StyledLink {...(lang && { lang })} inline={inline} href={href}>
```

Comment:
@simonsinclair since not all links will have the lang attribute we made a conscious decision not to add an empty lang prop for those cases ie `lang="" || lang=null` by conditionally passing the prop here

Reply:
That makes sense, Rowland, but `lang` would only be an empty string if explicitly set. Using `lang={lang}` will prevent the lang attribute from not being rendered when `lang === null`.